
2006-08-28  Stefan Siegl  <stesie@brokenpipe.de>

        * i386/linux/configure.ac: Add configuration option allowing to enable
        ISA-bus support in the pcmcia core.

        * i386/linux/configure: Regenerate.
        * i386/linux/device-drivers.h.in: Likewise.


Index: i386/linux/configure.ac
===================================================================
--- i386/linux/configure.ac.orig	2006-08-29 02:44:29.000000000 +0300
+++ i386/linux/configure.ac	2006-08-29 02:46:43.000000000 +0300
@@ -239,6 +239,24 @@ dnl PCMCIA device support.
 
 linux_DRIVER([i82365], [I82365], [i82365], [pcmcia])
 
+AC_DEFUN([AC_PCMCIA_OPTION], [
+  AC_ARG_ENABLE([pcmcia-$1], 
+  AS_HELP_STRING([--enable-pcmcia-$1], [$4]), [
+  if test "x$enableval" != "xno"; then
+    AC_DEFINE([CONFIG_$2], [1], [$4])
+    device_drivers="$device_drivers [$3]"
+    if test "${driver_class_pcmcia_selected+set}" != set; then
+      driver_class_pcmcia_selected=yes
+      AC_DEFINE_UNQUOTED([$driver_class_pcmcia_option], [1])
+      device_drivers="$device_drivers $driver_class_pcmcia_files"
+    fi
+  fi
+])])
+
+AC_PCMCIA_OPTION([isa], [ISA], [],
+		 [enable isa-bus-support (pcmcia core)])
+
+
 dnl PCMCIA device drivers.
 
 linux_DRIVER([3c574_cs], [3C574_CS], [3c574_cs], [pcmcia])
